zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/usermanagement/RoleController.java

```diff
@@ -369,4 +369,17 @@ private CompletableFuture<ResponseEntity<Object>> removeMemberFromRole(
                 .withAuthentication(authenticationProvider.getCamundaAuthentication())
                 .removeMember(request));
   }
+
+  private boolean hasGroupClaimConfigured() {
```

Comment:
🙃 It's sort of what we want to know, in the end:
```suggestion
  private boolean areGroupsManagedExternally() {
```